papaparse.js

```diff
@@ -643,6 +644,20 @@ License: MIT
 			}
 
 			xhr.open(this._config.downloadRequestBody ? 'POST' : 'GET', this._input, !IS_WORKER);
+			if (this._config.download && this._config.download.timeout) {
+				timeout = this._config.download.timeout;
+				timeout = typeof timeout === 'number' ?
```

Comment:
I think it's simpler to directly call parseInt on the option and only use if is set.

Reply:
Done